packages/core/data-transfer/src/strapi/providers/utils.ts

```diff
@@ -18,7 +18,13 @@ interface IDispatchOptions {
 
 type Dispatch<T> = Omit<T, 'transferID' | 'uuid'>;
 
-export const createDispatcher = (ws: WebSocket) => {
+export const createDispatcher = (
+  ws: WebSocket,
+  retryMessageOptions = {
+    retryMessageMaxRetries: 5,
+    retryMessageTimeout: 15000,
+  }
```

Comment:
would it make sense to use an exponential backoff pattern ?
![image](https://github.com/strapi/strapi/assets/20578351/8b7f58af-2789-4f17-b4ed-0f87d9699aaa)

15 seconds seems a bit too much if the connection was lost for a split second.

Reply:
not blocking the PR at all 😄 was reading this and wondered if it would be a nice addition for the future

Reply:
This isn't actually a "retry on failure" it's a "retry if we've been waiting too long" and that's where the 15 seconds comes in. Essentially what is happening here is that we can't detect in every case if a message is actually lost, the best we can do it wait 15 seconds to see if we get a reply and if we don't we try again.

Reply:
That makes sense 💡thank you!

Reply:
We send very long messages about 1mb so we do expect it to take time to send a single message a 100ms is too quick